src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -73,6 +74,28 @@ const ComposeForm = props => {
     draftAutoSaveTimeout,
   );
 
+  const localStorageValues = {
+    atExpires: localStorage.atExpires,
+    hasSession: localStorage.hasSession,
+    sessionExpiration: localStorage.sessionExpiration,
+    userFirstName: localStorage.userFirstName,
+  };
+
+  const resetUserSession = () => {
+    const timeout = setTimeout(() => {
+      Object.keys(localStorageValues).forEach(storageItem => {
+        if (!localStorage.getItem(storageItem)) {
+          localStorage.setItem(storageItem, localStorageValues[storageItem]);
+        }
+      });
+    }, 1000);
+    setTimeoutId(timeout);
+  };
+
+  const noTimeout = () => {
+    clearTimeout(timeoutId);
+  };
```

Comment:
This looks great! Just to note that all of us should start adding func like these into helper files to save space when we can.

Reply:
Great idea!  I'll move it to a helper file